inst-support-eng/panda-pto
# Rspecs

to test

- pull branch locally.
- run `bundle install` to update gems 
- in branch terminal run `rspec` tests should take about 5 ~ 6 minutes to run
-- note, some controllers will only have print statements as they currently are placeholders for future features

No UI changes should have occurred during implementing these tests, I ran through testing most of the selenium / feature tests manually as well so they should work as intended, but double check

DEPLOY: 

Add ENV['FEEDBACK_FORM'] to heroku before deploy to avoid sads

This closes #188 closes #164 and milestone 16.
